targets/ramips-mt7621

```diff
@@ -8,7 +8,9 @@ device d-link-dir-860l-b1 dir-860l-b1
 
 ## ZBT
 
-device zbt-wg3526 zbt-wg3526
+device zbt-wg3526-16m zbt-wg3526-16M
+factory
```

Comment:
please also add a line like this after factory, before the 32M line:
manifest_alias zbt-wg3526
this ensures, that people already running this device will keep getting updates via an autoupdater
(idea stolen from the gluon next branch)

Reply:
could not upgrade on a new build:
```
sysupgrade /tmp/firmware.bin  -F
----
Sysupgrade is not yet supported on zbt-wg3526.
Image check 'platform_check_image' failed.
```

Reply:
@rotanid I'm not sure whether that is a good idea, as there are also 16Ms out there which have zbt-wg3526 as their device profile at the moment, aren't they? Maybe it makes more sense for the ac1200pro?
The 16M-firmware also runs on the 32M-devices (maybe that's genofire's issue - someone ran the 16M version on a 32M device and it seemed to work).

I have used the lowercase "M" as the sanitize_image_name-function in libplatforminfo which is used by the get_image_name-function makes everything lowercase and strips non-alphanumeric characters, but maybe I'm telling you old news.

@genofire That doesn't have to do anything with what rotanid said, I think. 

You are running a firmware based on an old commit. Please flash the new one with tftp if you can.
Can you please execute following commands and post the output here?
```
lua -e 'print(require("platform_info").get_image_name())'
cat /tmp/sysinfo/board_name
cat /tmp/sysinfo/model
cat /proc/cpuinfo
```

Have you tried both (the 16M and the 32M-image)? I suspect you installed a 16M-image or there was a problem with force before. The strange thing in your case is that the image check magic for both routers is the same (but the partitioning is different of course). There is no check for the board-name on sysupgrade that I know. You shouldn't get the "Sysupgrade is not yet supported on zbt-wg3526." error if you used any of my commits. They were always consistent regarding platform.sh and ramips.sh. Best start with a clean buildtree.

If you can't use tftp or it doesn't even work after a fresh tftp flash try the following:
If the model-file gives you ZBT-WG3526 (16M) or ZBT-WG3526 edit the tmp-file and change it to ZBT-WG3526 (32M).
Edit the board_name-file and change it to zbt-wg3526-32M.
Try sysupgrade with the 32M-image.
If that doesn't work, try it the other way round:
Edit the tmp-file and change it to ZBT-WG3526 (16M).
Edit the board_name-file and change it to zbt-wg3526-16M.
Try sysupgrade with the 16M-image.

Reply:
@CodeFetch we won't support peoples custom builds or people flashing a wrong firmware (16m fw on 32m device) with gluon, we never do. if we start doing that, we end in maintenance hell - or something similar ;)
we do support however people already using the 16M device with the old naming, therefore we need the manifest_alias for the 16M device, as i already wrote.
i didn't intend to criticize the small "m" btw!

Reply:
@genofire try `sysupgrade -F $file` instead of `sysupgrade $file -F`

Reply:
Works now ^^ - sry think it was an old build:
(for update i have add a line with `zbt-wg3526` in `/lib/upgrade/platform.sh` - which disappear after update).
```
# grep zbt-wg3526 /lib/upgrade/platform.sh
	zbt-wg3526-16M|\
	zbt-wg3526-32M|\
# cat /tmp/sysinfo/board_name
zbt-wg3526-32M
```
@corny sry i forget.